modules/iam-policy-management/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/IamPolicyManagement.java

```diff
@@ -53,7 +56,7 @@
 
   public static final String DEFAULT_SERVICE_NAME = "iam_policy_management";
 
-  public static final String DEFAULT_SERVICE_URL = "https://iam.cloud.ibm.com";
+  public static final String DEFAULT_SERVICE_URL = "https://iam.test.cloud.ibm.com";
```

Comment:
We should be using the production version of the API definition for SDK generation.

Reply:
Yes, need to fix that @padamstx . Will get that fixed and then make the PR ready for review and out of draft mode. Thanks